src/webvr-polyfill.js

```diff
@@ -100,7 +100,7 @@ WebVRPolyfill.prototype.enablePolyfill = function() {
   navigator.getVRDisplays = this.getVRDisplays.bind(this);
 
   // Polyfill native VRDisplay.getFrameData
-  if (this.nativeWebVRAvailable && this.isCardboardCompatible() && window.VRFrameData) {
+  if (this.nativeWebVRAvailable && window.VRFrameData) {
```

Comment:
Sorry, I missed this...

Why do you think we need that check? The code is already inside a conditional for `!('VRFrameData' in window)`. Is there a case where `VRFrameData` will not exist but `getFrameData` will?